docs/docs/tutorials/datapane-cloud/projects.md

```diff
@@ -1,6 +1,5 @@
-# Projects
-
-## Projects
+!!! info
+If you are on the Cloud Starter plan, you will be limited to a single Project, and won't be able to create another until you upgrade to Cloud Pro.
```

Comment:
Admonition is broken

<img width="824" alt="Screenshot 2022-10-21 at 14 25 13" src="https://user-images.githubusercontent.com/15690380/197206636-388a5a55-56a2-4824-849f-2b1e4665b5e4.png">